scipy/stats/_continuous_distns.py

```diff
@@ -3867,6 +3867,59 @@ def _pdf(self, x, a, b):
         sq = np.hypot(1, x)  # reduce overflows
         return fac1 * sc.k1e(a * sq) * np.exp(b*x - a*sq) / sq
 
+    def _sf(self, x, a, b):
+        if np.isscalar(x):
+            # If x is a scalar, then so are a and b.
```

Comment:
why is that the case? does broadcasting happen before _sf is called?

Reply:
The `argsreduce` call inside `rv_continuous.sf()` does the broadcasting before the call to `_sf()` is made.